src/pages/events/eventsInfo/ongoingEvent/eventTable/TableContent.tsx

```diff
@@ -0,0 +1,182 @@
+import React, {useState} from 'react'
+import {Paper, Table, TableBody, TableCell, TableContainer, TableRow} from '@mui/material'
+import Report from './Report'
+import TableHeader from './TableHeader'
+import {IrowsInfo, Order} from '../../../../../types/types'
+
+const rows: IrowsInfo[] = [
+  {
+    status: 'Upcoming',
+    name: 'Quarter Evoluation',
+    date: 'Good Date',
+    evaluators: 'Evaluators',
+    evaluates: 'Evaluatees',
+  },
+  {
+    status: 'apcoming',
+    name: 'cuarter Evoluation',
+    date: 'Date',
+    evaluators: 'Evaluators',
+    evaluates: 'Evaluatees',
+  },
+  {
+    status: 'vpcoming',
+    name: 'cuarter Evoluation',
+    date: 'Date',
+    evaluators: 'Evaluators',
+    evaluates: 'dEvaluatees',
+  },
+  {
+    status: 'vpcoming',
+    name: 'cuarter Evoluation',
+    date: 'Date',
+    evaluators: '6Evaluators',
+    evaluates: 'Evaluatees',
+  },
+  {
+    status: 'apcoming',
+    name: 'cuarter Evoluation',
+    date: 'Date',
+    evaluators: 'cEvaluators',
+    evaluates: 'Evaluatees',
+  },
+  {
+    status: 'apcoming',
+    name: 'cuarter Evoluation',
+    date: 'Date',
+    evaluators: 'Evaluators',
+    evaluates: 'Evaluatees',
+  },
+
+  {
+    status: 'apcoming',
+    name: 'cuarter Evoluation',
+    date: 'fdDate',
+    evaluators: 'asdEvaluators',
+    evaluates: 'zEvaluatees',
+  },
+  {
+    status: 'apcoming',
+    name: 'cuarter Evoluation',
+    date: 'zDate',
+    evaluators: 'dfgdsEvaluators',
+    evaluates: 'zEvaluatees',
+  },
+  {
+    status: 'apcoming',
+    name: 'cuarter Evoluation',
+    date: 'zDate',
+    evaluators: 'dfgdsEvaluators',
+    evaluates: 'zEvaluatees',
+  },
+  {
+    status: 'apcoming',
+    name: 'cuarter Evoluation',
+    date: 'zDate',
+    evaluators: 'dfgdsEvaluators',
+    evaluates: 'zEvaluatees',
+  },
+  {
+    status: 'apcoming',
+    name: 'cuarter Evoluation',
+    date: 'zDate',
+    evaluators: 'dfgdsEvaluators',
+    evaluates: 'zEvaluatees',
+  },
+  {
+    status: 'apcoming',
+    name: 'cuarter Evoluation',
+    date: 'zDate',
+    evaluators: 'dfgdsEvaluators',
+    evaluates: 'zEvaluatees',
+  },
+  {
+    status: 'apcoming',
+    name: 'cuarter Evoluation',
+    date: 'zDate',
+    evaluators: 'dfgdsEvaluators',
+    evaluates: 'zEvaluatees',
+  },
+
+  {
+    status: 'apcoming',
+    name: 'cuarter Evoluation',
+    date: 'zDate',
+    evaluators: 'dfgdsEvaluators',
+    evaluates: 'zEvaluatees',
+  },
+]
+
+const TableContent = () => {
+  const [orderDirection, setOrderDirection] = useState<Order>('asc')
+  const [valueToOrderBy, setValueToOrderBy] = useState<keyof IrowsInfo>('status')
+
+  const handleRequestSort = (event: React.MouseEvent<unknown>, property: keyof IrowsInfo) => {
+    const isAsc = valueToOrderBy === property && orderDirection === 'asc'
+    setOrderDirection(isAsc ? 'desc' : 'asc')
+    setValueToOrderBy(property)
+  }
+
+  function descendingComporator<T>(a: T, b: T, orderBy: keyof T) {
```

Comment:
Don't use this kind of functions directly in component, keep all components clear
Make utils for this kind of functions